MarvelData/Presentation/ViewController/ComicDetails/ComicDetailsViewController.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  ComicDetailsViewController.swift
+//  MarvelData
+//
+//  Created by Swapnali Patil on 05/05/22.
+//
+
+import UIKit
+import SDWebImage
```

Comment:
let say tomorrow we want to remove SDWebImage and use another library, like Kingfisher or something, how would you achieve it? 
Make protocol and inject it here and have the image download implementation somewhere else. 
so in that way we would make change at one place if we want to move to the another library